packages/components/chip/src/Chip.test.tsx

```diff
@@ -1,27 +1,232 @@
-import { render, screen } from '@testing-library/react'
+import { Icon } from '@spark-ui/icon'
+import { InfoFill } from '@spark-ui/icons'
+import { fireEvent, render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import { describe, expect, it, vi } from 'vitest'
 
 import { Chip } from './Chip'
+import { ChipClearButton } from './ChipClearButton'
+import { ChipContent } from './ChipContent'
+import { ChipLeadingIcon } from './ChipLeadingIcon'
 
 describe('Chip', () => {
   it('should render', () => {
-    render(<Chip>Hello World!</Chip>)
+    // Given
+    const props = { children: 'Hello World!' }
+
+    render(<Chip {...props} />)
+
+    expect(screen.getByText(props.children)).toBeInTheDocument()
+  })
+
+  it('should be visible', () => {
```

Comment:
Aren't those two tests redundant? (`it should render` and `it should be visible`)